### ISSUE TITLE
Config parameters for entries per page in the vol metadata

### ISSUE THREAD

**Issue by wimpers** [openvstorage/framework#1050]
- default should be 64 (METADATASTORE_BITS = 6).
- Make sure MD cache size calculation is taking this into account.

in case of questions ... check with @saelbrec

**Comment by khenderick** [comment]
`METADATASTORE_BITS` is an env variable, should be set to 6 iso 8. 
metadata_page_capacity = capacity of X bits.

**Comment by khenderick** [comment]
Finally fixed by #1055, packaged in openvstorage-2.7.4-rev.4164.e5fcb6d

**Comment by JeffreyDevloo** [comment]
### Steps
- Setup latest environment
- Configured latest environment
- Checked etc file for METADATASTORE_BITS
- Checked current 

### Output
#### Service file
```
[Unit]
Description=Open vStorage StorageDriver
Requires=ovs-albaproxy_myvpool01.service 
Wants=ovs-watcher-volumedriver.service
After=ovs-watcher-volumedriver.service

[Service]
Type=simple
Environment=METADATASTORE_BITS=6
ExecStartPre=/bin/bash -c "if [ ! -d /opt/OpenvStorage/run ]; then mkdir /opt/OpenvStorage/run; chown ovs:ovs /opt/OpenvStorage/run; fi; touch /opt/OpenvStorage/run/storagedriver_myvpool01.lock"
ExecStart=/usr/bin/volumedriver_fs.sh -f --config arakoon://config/ovs/vpools/ad0afa7a-fe84-4f0e-8e37-113708b28513/hosts/myvpool01mdbttNTebUp6umVI/config?ini=%2Fopt%2FOpenvStorage%2Fconfig%2Farakoon_cacc.ini --lock-file /opt/OpenvStorage/run/storagedriver_myvpool01.lock --logrotation --mountpoint /mnt/myvpool01 --logsink console: -o big_writes -o sync_read -o allow_other -o use_ino -o default_permissions -o uid=1001 -o gid=1001 -o umask=0002
ExecStop=/bin/bash -c "if mount | grep /mnt/myvpool01; then fusermount -u /mnt/myvpool01; fi "
ExecStopPost=/bin/bash -c "for i in `seq 0 9`; do if mount | grep /mnt/myvpool01; then sleep 10s; else break; fi; done; if mount | grep /mnt/myvpool01; then umount -l /mnt/myvpool01; ps xa |grep volumedriver_fs |grep -v grep |awk '{print $1}' |xargs kill; fi"
Restart=on-failure
RestartSec=360
TimeoutStopSec=30
LimitMEMLOCK=infinity

[Install]
WantedBy=multi-user.target
```
#### Cache size test
```
# Copyright (C) 2016 iNuron NV
#
# This file is part of Open vStorage Open Source Edition (OSE),
# as available from
#
#      http://www.openvstorage.org and
#      http://www.openvstorage.com.
#
# This file is free software; you can redistribute it and/or modify it
# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
# as published by the Free Software Foundation, in version 3 as it comes
# in the LICENSE.txt file of the Open vStorage OSE distribution.
#
# Open vStorage is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY of any kind.
from ovs.extensions.storageserver.storagedriver import StorageDriverConfiguration
from ovs.dal.lists.vdisklist import VDiskList


class MDSTest(object):

    @staticmethod
    def test():
        test_passed = False
        for vdisk in VDiskList.get_vdisks():
            storagedriver_config = StorageDriverConfiguration('storagedriver', vdisk.vpool_guid, vdisk.storagedriver_id)
            storagedriver_config.load()
            metadata_page_capacity = 2 ** 6
            cluster_size = storagedriver_config.configuration.get('volume_manager', {}).get('default_cluster_size', 4096)
            # x bytes are reserved for mds cache
            cache_capacity = int(min(vdisk.size, 2 * 1024 ** 4) / float(metadata_page_capacity * cluster_size))
            # Expecting 4096 for 1gb disks:
            assert cache_capacity == 4096, 'Cache capacity is not what I expected'
            test_passed = True

        if test_passed == True:
            print 'Test passed'


if __name__ == "__main__":
    MDSTest.test()
Test passed
```

### Test result
Test passed.

### Packages
```
-openvstorage                         2.7.6-rev.4301.8e25118-1        amd64        openvStorage
-openvstorage-backend                 1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin
-openvstorage-backend-core            1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin core
-openvstorage-backend-webapps         1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin Web Applications
-openvstorage-core                    2.7.6-rev.4301.8e25118-1        amd64        openvStorage core
-openvstorage-hc                      1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin HyperConverged
-openvstorage-sdm                     1.6.6-rev.437.ad174b6-1         amd64        Open vStorage Backend ASD Manager
-openvstorage-webapps                 2.7.6-rev.4301.8e25118-1        amd64        openvStorage Web Applications

```
